tests/console/nvme_checks.pm

```diff
@@ -0,0 +1,113 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Smoke tests for NVM Express.
```

Comment:
smoke tests also validate or verify something. i dont get that part i think. if the problem is accurate description it is important but i believe it does not make any difference here

Reply:
it is just about the name of the file, really minor thing.

Reply:
ok. i get it. I picked nvme_checks.pm because i was thinking that these set of tests are some basic checks. To not repeat `validation_*` in case that we come up with another testcase later that actually do some validation. I am resolving this if you are ok. feel free to reopen it if you want